src/js/components/config/team-config.js

```diff
@@ -0,0 +1,14 @@
+const TeamConfig = [
+  {
+    name: 'T Hansen',
+    path: 'https://thomasandrewhansen.com',
+    imgSrc: 'https://avatars.githubusercontent.com/u/45204791?s=56&v=4',
+  },
+  {
+    name: 'J Darlow',
+    path: 'https://www.producthunt.com/posts/shred-index',
+    imgSrc: 'https://avatars.githubusercontent.com/u/1634995?s=60&u=ebaca76ccf999d03b9259721d800571f6df7a6a4&v=4',
```

Comment:
```suggestion
    name: 'jakxnz',
    path: 'https://github.com/jakxnz',
    imgSrc: 'https://avatars.githubusercontent.com/u/1634995?s=60&u=ebaca76ccf999d03b9259721d800571f6df7a6a4&v=4',
```

I'm not picky, but while I'm here 😂

Reply:
Also, is the 60px size enough for retina? Or do we want to go for 120 to be safe?